src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -221,17 +237,7 @@ synchronized void loadSidelines() {
         // If we haven't spun up a VirtualSpout yet, we create it here.
         if (fireHoseSpout == null) {
             // Create the main spout for the namespace, we'll dub it the 'firehose'
-            fireHoseSpout = new VirtualSpout(
-                // We use a normal virtual spout identifier here rather than the sideline one because this is NOT a sideline,
-                // it's our firehose.
-                fireHoseIdentifier,
-                getSpoutConfig(),
-                topologyContext,
```

Comment:
I like this approach.  Do we need to store a copy of topologyContext anymore?

Reply:
Probably not. I don’t think we use it. Should we just drop it?

Reply:
It's used to construct `ConsumerPeerContext`, which should maybe just be moved to a constructor parameter? Not sure.  I started stripping topologyContext out, I'll stick it in a branch in case we want to revisit but I don't think we should sweat it right now.

Reply:
oh its just used in open right, i just meant dropping saving the reference in the class beyond the scope of where its used.  Not sure it matters much tho